tests/e2escenarios/core_beta_flow.go

```diff
@@ -70,20 +73,20 @@ var _ = Describe("Core beta flow", func() {
 			//helper.FileShouldContainSubstring(configFile, "SourceLocation: .")
 			helper.FileShouldContainSubstring(configFile, "Project: "+project)
 
-			configSession := helper.CmdShouldPass(odo, append([]string{"config", "set", "--env", "FOO=bar"}, extraArgs...)...)
+			configSession := helper.CmdShouldPass(strings.TrimSpace(fmt.Sprintf("odo config set --env FOO=bar " + argString)))
```

Comment:
why `TrimSpace` and not writing it as an array directly?

Reply:
```
Running odo component create java mycomponent --app myapp 
[odo] Error: accepts between 0 and 2 arg(s), received 3
[odo] Usage:
[odo]   odo component create <component_type> [component_name] [flags]
[...]
```
This error is reported when i do not trim the space. It seems https://github.com/openshift/odo/pull/1612/files#diff-66f499c995fc950f403a939757588820R13 do not accept blank argument at the end of command string.

Reply:
Ah sorry, my questions were supposed to be, why was this changed from passing array to string. 
There are some problems with this approach. See my comment for `CmdParser` function

Reply:
https://github.com/openshift/odo/pull/1612#discussion_r280374724